tests/unit/model/session_test.py

```diff
@@ -19,7 +28,7 @@
 def test_str() -> None:
     assert_equal(
         str(
-            Session(
+            UnprovenSession(
```

Comment:
`Session` could be used directly in all instances of this test file.

Reply:
Reverted in d73322c16d65cf8838334a7916105f42e4e0f9ec.